### PR TITLE
Add payments endpoint

### PR DESCRIPTION
**Pull request by obohatov**

### CODE REVIEW

File: payments/tests.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+from rest_framework import status
+
+from payments.models import Payment
+from borrowings.models import Borrowing
+from books.models import Book
+
+
+class BaseTest(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(
```
Comment (eduardhabryd) [inline payments/tests.py:16]: use `get_user_model() `instead of `User.objects.`

File: payments/tests.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        self.book = Book.objects.create(
+            title="Some title",
+            author="Some author",
+            cover=Book.Covers.HARD,
+            inventory=10,
+            daily_fee=5.00
+        )
+        self.borrowing = Borrowing.objects.create(
+            user_id=self.user,
+            book_id=self.book,
```
Comment (eduardhabryd) [inline payments/tests.py:32]: here should be:

```
 user=self.user,
 book=self.book,
```

**Review by eduardhabryd — CHANGES_REQUESTED** [review]

File: payments/admin.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (eduardhabryd) [inline payments/admin.py:3]: register payment model here

**Review by eduardhabryd — CHANGES_REQUESTED** [review]

**Review by eduardhabryd — APPROVED** [review]
Good Job!

**Review by CaCtUs1k — APPROVED** [review]

**Pull request merged**